src/Templates/MLDotNet/Microsoft.VisualStudio.Templates.MLDotNet/Content/ConsoleApp/Program.cs

```diff
@@ -0,0 +1,87 @@
+/* This template shows the building blocks for training a machine learning model with ML.NET (https://aka.ms/mlnet).
+ * This model predicts whether a sentence has a positive or negative sentiment. It is based on a sample that can be 
+ * found at https://aka.ms/mlnetsentimentanalysis, which provides a more detailed introduction to ML.NET and the scenario. */
+
+using System;
+using System.IO;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Transforms.Text;
+
+namespace $safeprojectname$
+{
+    class Program
+    {
+        static void Main()
+        {
+            // 1. Build an ML.NET pipeline for training a sentiment analysis model.
+            Console.WriteLine("Training a model for Sentiment Analysis using ML.NET");
+            var mlContext = new MLContext();
+            // 1a. Load the training data using a TextLoader.
+            var reader = 
+                new TextLoader(
```

Comment:
This and the next line can be simplified to follow the pattern below:

```
IDataView trainingData = mlContext.Data.ReadFromTextFile(
                new[]
                {
                    new TextLoader.Column("SepalLength", DataKind.R4, 0),
                    new TextLoader.Column("SepalWidth", DataKind.R4, 1),
                    new TextLoader.Column("PetalLength", DataKind.R4, 2),
                    new TextLoader.Column("PetalWidth", DataKind.R4, 3),
                    new TextLoader.Column("Label", DataKind.Text, 4),
                },
                "iris-data.txt",
                options => options.Separator = ",");
```

Reply:
Thanks! Updated